src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs

```diff
@@ -21,7 +21,46 @@ public class PackageManagerSearchElementViewModel : BrowserItemViewModel, IEquat
         public ICommand VisitRepositoryCommand { get; set; }
         public ICommand DownloadLatestToCustomPathCommand { get; set; }
 
-        public new PackageManagerSearchElement Model { get; internal set; }
+        /// <summary>
+        /// The PackageManagerSearchElement Model this VM links to
+        /// </summary>
+        public PackageManagerSearchElement SearchElementModel { get; internal set; }
```

Comment:
so this is really the model?

Reply:
This is the Model model .. yes :/ I came up with this name to avoid the Ambiguity.